### ISSUE TITLE
Game Theory considerations

### ISSUE THREAD

**Issue by jonas4climate** [chrismbryant/amazon-confidence-interval#15]
This was just a thought, not sure if it is too relevant on the current scale, but wanted to get it out there.

If this extension were to become popular and be used by a large number of amazon users, it would be possible that everyone choosing the same recommended seller will overload the seller's logistics capacity to deliver, stock being available etc. and would therefore lead to a worse overall experience. 

I assume that this would balance out if reviews come in significantly quick because the extension would then recommend a different seller but this might be something worth to consider? Especially if multiple sellers are almost identically recommendable.

**Comment by musicin3d** [comment]
Good question. I believe amazon tracks quantity in stock, so it ought to prevent sales when a seller runs out.

**Comment by jonas4climate** [comment]
Yeah, I didn’t mean that they would sell more than exists in stock. I meant
that they will have longer down-time i.e. time without stock as a specific
seller would receive more purchases than usual until out of stock.

But to be honest, this is probably not of much concern as long as the
extension isn’t used by millions of users.

**Comment by musicin3d** [comment]
I think you're right about it balancing out though. This is armchair conjecture, but IMO if there is such demand and another seller can handle the volume better, then they deserve to come out on top. Counterpoint to that could be, "What if _none_ of the sellers can handle it?" I think the dropping ratings across all listings would expose an opportunity for another seller who _can_ handle the volume. 

I have only a cursory knowledge of game theory. Which principle applies to this situation?

**Comment by chrismbryant** [comment]
Interesting point. I'm not sure we're really _recommending_ one seller over another here, we're just providing some perspective on one aspect of the reliability of Amazon's star rating. I don't think that the results from our extension would dictate a person's purchase behavior too deterministically, so I'm not really worried about dramatically changing the landscape of the Amazon marketplace.

Anyway, I would be surprised if Amazon didn't already take this into consideration internally when designing their recommendation engine to choose what results to show to what users and in what order to display those results. I wonder if the products marked "Amazon's Choice" are already affected by the dynamics you brought up, @j0ner0n...
